### ISSUE TITLE
Clickable items not animated

### ISSUE THREAD

**Issue by DanielNovak** [emilsjolander/StickyScrollViewItems#4]
When you have a button inside the sticky view and some other clickable items below the sticky view and you click on the button, then the click is registered, but the button background drawable is almost never changed to clicked/pressed state.

**Comment by lukesleeman** [comment]
Yep, I can confirm this is still happening.  Does anybody have a fix?
